components/server/src/data/datamodel.json

```diff
@@ -5113,6 +5136,25 @@
                             "type": "integer"
                         }
                     ]
+                },
+                "missing_metrics": {
+                    "name": "Metric type",
+                    "name_plural": "Metric types",
+                    "attributes": [
+                        {
+                            "name": "Subject type",
+                            "key": "subject_type"
+                        },
+                        {
+                            "name": "Metric type",
+                            "key": "metric_type"
+                        },
+                        {
+                            "name": "Missing from reports",
+                            "key": "reports",
+                            "url": "report_url"
```

Comment:
The URL isn't filled in by the collector. And it wouldn't work anyway, as this is a list of reports and we have no support in the frontend to add URLs to a comma-separated string. 
```suggestion
```